projects/step-frontend/src/lib/modules/timeseries/util/oql-builder.ts

```diff
@@ -36,6 +36,6 @@ export class OQLBuilder {
   }
 
   build(): string {
-    return this.clauses.join(this.separator);
+    return this.clauses.join(' ' + this.separator + ' ');
```

Comment:
separator is set in the open method, I think this change will break something

Reply:
it is also initialize with 'and'

Reply:
this is not correct tho, I can see that the ```OQLBuilder``` is used only here

<img width="571" alt="image" src="https://github.com/exense/step-frontend/assets/8954749/c73ac2ed-8c05-49e6-98cb-d580c58de585">

however, keep in mind we should be able to unit test each piece individually, so consider the following scenario

```const foo = new OQLBuilder().open('-').append('a').append('b').build();```

now, when I try to test it

```expect(foo).toBe('a-b');```

the test will fail, because the actual result will be ```'a - b'```

Reply:
anyway, up to u to decide, I would personally leave the user provide the proper separator from the outside